homeassistant/scripts/check_config.py

```diff
@@ -247,6 +248,7 @@ def mock_package_error(  # pylint: disable=unused-variable
         res['secret_cache'] = dict(yaml.__SECRET_CACHE)
     except Exception as err:  # pylint: disable=broad-except
         print(color('red', 'Fatal error while loading config:'), str(err))
+        res['except_global'].append(err)
```

Comment:
There will always only be one of these, so instead of adding a new key, simply add it to "General Errors" `res['except'].setdefault(ERROR_STR, []).append(err)`